packages/usa-file-input/src/index.js

```diff
@@ -172,21 +170,42 @@ const createTargetArea = (fileInputEl) => {
  */
 const createVisibleInstructions = (fileInputEl) => {
   const fileInputParent = fileInputEl.closest(DROPZONE);
-  const itemsLabel = getItemsLabel(fileInputEl);
+  const acceptsMultiple = fileInputEl.hasAttribute("multiple");
   const instructions = document.createElement("div");
-  const dragText = `Drag ${itemsLabel} here or`;
-  const chooseText = "choose from folder";
+  const customDragText = fileInputEl.dataset.dragText;
+  const customDragTextSingular = fileInputEl.dataset.dragTextSingular;
```

Comment:
## Specifying singular and plural

How do you feel about changing `customDragText` to `customDragTextPlural` to keep things specific?

I think this would follow file input defaults since the multiple variant requires a multiple attribute.